modules/ossm-vs-istio.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/understanding-ossm.adoc
+
+[id="ossm-vs-istio_{context}"]
+= Comparing {ProductName} and upstream Istio community installations
+
+An installation of {ProductName} differs from upstream Istio community installations in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift.
+
+The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
+
+== Automatic injection
```

Comment:
Add explicit anchors throughout this module.

Reply:
Fixed.